src/features/bivariate_manager/readme.md

```diff
@@ -0,0 +1,52 @@
+## Bivariate Manager
+
+Bivariate manager is a tool for analyzing and finding new insights from a set of suggested layers.
+
```

Comment:
I guess it would be usefull to add few words about what is *numerators* and *denominators* in cotext of bivariate manager

Reply:
done